src/preset.ts

```diff
@@ -449,11 +449,11 @@ export function setOriginalUserPresetsFile(folder: string, presets: PresetsFile
     originalUserPresetsFiles.set(folder, presets);
 }
 
-export function setPresetsFile(folder: string, presets: PresetsFile | undefined) {
-    presetsFiles.set(folder, presets);
+export function setExpandedPresetsFile(folder: string, presets: PresetsFile | undefined) {
```

Comment:
Just confirming, this is just a rename to make it more obvious about how we're saving the state of these files right?

Reply:
yup!